### PR TITLE
protobuf 3.7.1 and cpp grpc 1.13.1

### PR DESCRIPTION
**Pull request by evie404**
It's good to upgrade and test the repo against newer versions. Helps us with Bazel upgrades too since those repos have their own Bazel build files.

These are versions I can upgrade without significant changes, so sticking with those for now. We're still very far behind HEAD unfortunately.

build passed in: https://travis-ci.org/vsco/rules_protobuf/builds/586064489

### CODE REVIEW

File: protobuf/internal/proto_repositories.bzl [hunk 1]
```diff
@@ -6,11 +6,14 @@ def proto_repositories(excludes = [],
...
                        protobuf_deps = DEPS,
                        protobuf_requires = [
+                         "bazel_skylib",
                          "com_google_protobuf",
                          "protobuf",
                          "protobuf_clib",
                          "protocol_compiler",
                          "protobuf_headers",
+                         "zlib",
+                         "com_github_madler_zlib",
```
Comment (akakitani) [inline protobuf/internal/proto_repositories.bzl:16]: Did zlib end up being required for all the language implementations with 3.7.1?

Comment (evie404) [reply protobuf/internal/proto_repositories.bzl:16]: zip ended up being a protobuf dep instead of a grpc one i think

**Review by akakitani — APPROVED** [review]
👍  would we be able to move our internal repos to use 3.7.1 after we merge this?

**Comment by evie404** [comment]
@akakitani we're already on 3.7.1 for some repos afaik. but yeah, this would give even more assurance

**Pull request merged**